cupy_setup_build.py

```diff
@@ -1004,7 +1004,17 @@ def _nvcc_gencode_options(cuda_version):
         #
         #   https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#options-for-steering-gpu-code-generation
 
-        if cuda_version >= 11000:
+        if cuda_version >= 11040:
+            arch_list = ['compute_35',
+                         'compute_50',
+                         ('compute_60', 'sm_60'),
+                         ('compute_61', 'sm_61'),
+                         ('compute_70', 'sm_70'),
+                         ('compute_75', 'sm_75'),
+                         ('compute_80', 'sm_80'),
+                         ('compute_86', 'sm_86'),
+                         'compute_86']
```

Comment:
This entry is intended as a fallback for all `compute_8X`.
```suggestion
                         'compute_80']
```

Reply:
I think GPU with cc=80 and cc=86 are covered by the two lines above, and the line is for forward compatibility for future cc>86. Also I think nVidia doesn’t insert a new cc number below the maximum one. Do I have some misunderstanding?

Reply:
Ah, you are correct, that seems like a correct approach to provide forward compatibility 😄 
https://github.com/NVIDIA/cuda-samples/blob/v11.4/Samples/vectorAddMMAP/Makefile#L323-L328